packages/cli/package.json

```diff
@@ -18,15 +18,15 @@
     "mud2": "./dist/mud2.js"
   },
   "scripts": {
-    "build": "pnpm run build:js && chmod +x ./dist/mud.js",
-    "build:js": "vite build",
-    "codegen": "tsx ./scripts/codegen.ts",
+    "build": "pnpm run build:js",
+    "build:js": "vite build && chmod +x ./dist/mud.js ./dist/mud2.js",
+    "clean": "pnpm run clean:js",
+    "clean:js": "rimraf dist",
     "dev": "vite build --watch",
+    "generate-test-tables": "tsx ./scripts/generate-test-tables.ts",
```

Comment:
This looked more like a one-off script so I moved it out of the build path (doesn't need to be regenerated each time and is expected to be run locally and changes committed).

Reply:
I agree, though it's easy to forget to run this (and this one changes often, as it's meant to test any changes to tablegen), which is why it was part of `test`, even though it doesn't really fit there too well.
If not part of `test`, it would be nice to have it somewhere else in CI at least

Reply:
added back into `test` script for now, will refactor in a follow up as part of #637